webpack.config.js

```diff
@@ -10,37 +10,47 @@ const path = require('path');
 // creates index.html folder and puts it in dist folder
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const dotenv = require('dotenv-webpack');
+const LodashPlugin = require('lodash-webpack-plugin');
 const ZipPlugin = require('zip-webpack-plugin');
 const env = require('./config.js').env;
 const url = require('./config.js')[env].refocusUrl;
 const botName = require('./package.json').name;
-const Uglify = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 
 const config = {
+  mode: 'production',
```

Comment:
does this make it always treated as production ?

Reply:
Yeah, from a webpack perspective anyway. This was already the case since we never specified a mode in any of the npm scripts and it falls back to production.

Reply:
are you sure about that. i thought the reason for the if was that we wanted the non uglified version for dev in front end for debugging

Reply:
You're right. There was a flag before that only uglified when production was set in NODE_ENV, I've added something similar back in. Nice catch!